### DISCUSSION TITLE
Is it possible to download NCBI protein database instead of genome?

### DISCUSSION THREAD

**Discussion by enriiquee** [kblin/ncbi-genome-download#183] [Q&A]
Hi! 
Nice tool! I used it many times before to donwload large genome data sets, but now I need to download Bacteria protein database but I was not able to find an optimal application to do this. Is it possible to do this with your app? 

Thank you in advance.

**Comment by elizabethmcd** [comment]
I believe you can do this using `--format protein-fasta`?

**Comment by kblin** [comment]
Yes, what @elizabethmcd said.
